docs/source/onedal/algorithms/graphs/graph-algorithm.rst

```diff
@@ -0,0 +1,49 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. highlight:: cpp
+.. default-domain:: cpp
+
+.. /use the algorithm name/
+.. _alg_graph_algorithm:
+
+.. /use the algorithm name/
+===================================
+Graph Algorithm
+===================================
+
+.. /use the algorithm name/
+.. include:: ../../../includes/graphs/graph-algorithm-introduction.rst
+
+------------------------
+Mathematical formulation
+------------------------
+
+.. /write the mathematical description of the algorithm/
+
+---------------------
+Programming Interface
+---------------------
+
+.. /use the algorithm name/
+Refer to :ref:`API Reference: Graph Algorithm <api_graph_algorithm>`.
```

Comment:
From docbuild: `WARNING: undefined label: api_graph_algorithm`

I don't see this topic in API Reference.

Reply:
do you need this topic? I think it's the example I've created, I don't see you really using it for anything. It's probably should be deleted